### DISCUSSION TITLE
Support for cron

### DISCUSSION THREAD

**Discussion by PanderMusubi** [github-linguist/linguist#6245] [Classification] [answered]
Which language from https://github.com/github/linguist/blob/master/lib/linguist/languages.yml should be used for syntax highlighting of (lines of) cron files? Or is this not yet supported?

**Comment by lildude** [comment, marked as answer]
Linguist doesn't have specific support for crontab files, but you can probably get 'good enough' highlighting using `shell`:

```shell
# This is a crontab comment
2,6,10,14,18,22 * * * /path/to/your/cmd > /dev/null 2>&1
```
